zend_abstract_interface/interceptor/php8/interceptor.c

```diff
@@ -39,6 +39,9 @@ static inline bool zai_hook_memory_table_del(zend_execute_data *index) {
 }
 
 #if defined(__x86_64__) || defined(__aarch64__)
+# if defined(__GNUC__) && !defined(__clang__)
+__attribute__((no_sanitize_address))
+# endif
```

Comment:
GCC's asan doesn't handle this part right, or something?

Reply:
Yep. At least some versions of GCC. For instance, I can't get it to repro within my buster container. It seems to put some things onto the stack and expects them to be still there after the stack base swap.